src/platform/plugins/shared/ai_assistant_management/selection/server/plugin.ts

```diff
@@ -77,7 +86,7 @@ export class AIAssistantManagementSelectionPlugin
         optionLabels: {
           [AIAssistantType.Default]: i18n.translate(
             'aiAssistantManagementSelection.preferredAIAssistantTypeSettingValueDefault',
-            { defaultMessage: 'Observability and Search only (default)' }
+            { defaultMessage: envSpecificScope + ' only (default)' }
```

Comment:
Same as above.

As a side note: if you need to do string concatenation, use the template literals `${envSpecificScope} only (default)`. It won't work here but in general that reads better.